ci-operator/config/openshift/knative-client/openshift-knative-client-release-next.yaml

```diff
@@ -0,0 +1,29 @@
+tag_specification:
+  name: '4.0'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: release-next
+base_images:
+  base:
+    name: '4.0'
```

Comment:
4.1 ?

Reply:
do you generate these?  check in the eventin / serving repos, we generate those files

Reply:
I used the serving one, should bump to 4.1
https://github.com/openshift/knative-serving/blob/master/openshift/ci-operator/generate-ci-config.sh#L7

Reply:
fixed

Reply:
Still 4.0